source/installation_guide/upgrading/from-2.x-to-3.0.rst

```diff
@@ -0,0 +1,37 @@
+=====================================
+Upgrading Dovecot from 2.x to 3.0/2.4
+=====================================
+
+.. contents::
+   :depth: 1
+   :local:
+
+
+About version numbers
+=====================
+
+With 3.0 / 2.4 release Pro and CE release have different major version numbers.
+In documentation any reference to 3.0 applies to 2.4 as well.
```

Comment:
Or should it be: `In documentation any reference to 2.4 applies to 3.0 as well, and usually (but not always) vice versa.` ? Since 3.0 can have some stuff not in 2.4.